README.md

```diff
@@ -1 +1,13 @@
-# ccd-definition-processor
\ No newline at end of file
+# ccd-definition-processor
+## Overview
+CI tooling to support colaberative development of CCD UI thought the use of text based representation of definitons. Supporting textual comparison and change control.
+## Usage
+```
+update options <sheet name> ...
+```
+
+*_Options:_*
+
+* -f    path to xlsx file
+* -D    path to the sheets folder
+* -a    (optional) import all files in sheets folder, the list of sheet names is ignored
```

Comment:
IMO tool should walk through all worksheets by default with possible an option of extracting only specific sheet if we have a time.